velox/exec/tests/JoinFuzzer.cpp

```diff
@@ -673,9 +675,31 @@ void JoinFuzzer::verify(core::JoinType joinType) {
           continue;
         }
       }
-
-      LOG(INFO) << "Testing plan #" << i << " with spilling";
-      actual = execute(altPlans[i], true /*injectSpill*/);
+      auto runWithSpilling = [&]() {
+        LOG(INFO) << "Testing plan #" << i << " with spilling";
+        int count = 0;
+        while (true) {
+          try {
+            count++;
+            actual = execute(altPlans[i], true /*injectSpill*/);
```

Comment:
If fuzzer join failure is file path not found error, can we consider to add a file path existence check in
```
  AssertQueryBuilder builder(plan);
  std::shared_ptr<TempDirectoryPath> spillDirectory;
  if (injectSpill) {
    spillDirectory = exec::test::TempDirectoryPath::create();
    builder.config(core::QueryConfig::kSpillEnabled, "true")
        .config(core::QueryConfig::kAggregationSpillEnabled, "true")
        .config(core::QueryConfig::kTestingSpillPct, "100")
        .spillDirectory(spillDirectory->path);
  }
```
And also log out the created file path or list the content of spill file directory? Probably add these checks inside execute? Thanks!